Analysis/DataModel/include/Analysis/Jet.h

```diff
@@ -43,8 +57,36 @@ DECLARE_SOA_INDEX_COLUMN(Jet, jet);
 DECLARE_SOA_INDEX_COLUMN(Track, track);
 } // namespace constituents
 
-DECLARE_SOA_TABLE(JetConstituents, "AOD", "JETCONSTITUENTS",
-                  constituents::JetId, constituents::TrackId);
+DECLARE_SOA_TABLE(JetConstituents, "AOD", "CONSTITUENTS",
+                  constituents::JetId,
+                  constituents::TrackId);
 
 using JetConstituent = JetConstituents::iterator;
+
+namespace constituentssub
+{
+DECLARE_SOA_INDEX_COLUMN(Jet, jet);
+DECLARE_SOA_COLUMN(Pt, pt, float);
+DECLARE_SOA_COLUMN(Eta, eta, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_COLUMN(Energy, energy, float);
+DECLARE_SOA_COLUMN(Mass, mass, float);
+DECLARE_SOA_DYNAMIC_COLUMN(Px, px, [](float pt, float phi) { return pt * TMath::Cos(phi); });
```

Comment:
These should probably become expression columns in the next step if they are accessed more than once